nessie/src/test/java/org/apache/iceberg/nessie/TestNessieCatalog.java

```diff
@@ -169,4 +171,25 @@ public void testWarehouseLocationWithTrailingSlash() {
                 + "/"
                 + TABLE.name());
   }
+
+  @Override
+  @Test
+  public void listTablesInEmptyNamespace() {
+    Namespace ns = Namespace.of("ns");
+
+    if (requiresNamespaceCreate()) {
+      catalog().createNamespace(ns);
+    }
+
+    TableIdentifier table1 = TableIdentifier.of(Namespace.empty(), "table_1");
+    TableIdentifier table2 = TableIdentifier.of(ns, "table_2");
+
+    catalog().buildTable(table1, SCHEMA).create();
+    catalog().buildTable(table2, SCHEMA).create();
+
+    // TODO this lists all available tables and deviates from the behavior in other catalogs and
+    // should probably be fixed
+    Assertions.assertThat(catalog().listTables(Namespace.empty()))
```

Comment:
@snazy, @adutra, @dimas-b: Nessie doesn't support empty namespace creation. So, should we allow listing tables from empty namespace?

Reply:
But nessie supports creating a table/view withing the empty namespace, which is a valid use case. If creating a table/view in an empty namespace is supported, then listing should be supported as well

Reply:
An empty namespace is not an "entity", right? Why would we want to "create" it?

That said, creating tables in the "empty" / "root" namespace should work with Nessie, IIRC.